lib/Core/Memory.h

```diff
@@ -382,6 +378,7 @@ class PersistentState : public ObjectState {
      * set.
      */
     UpdateList rootCauseLocations;
+    UpdateList pendingRootCauseLocations;
```

Comment:
@BGR360, can you convince me that this doesn't miss bugs?

```
store x => <some pmem>
exit
```

If we have a pending and a full list, you wouldn't catch that single write with no flush or fence, right?

Reply:
Writes don't pend; only flushes do.

Reply:
Oh, right, I remember this now.